modules/DGM/PSO.cpp

```diff
@@ -63,58 +52,7 @@ void DirectGraphicalModels::PSO::reset() {
     std::fill(m_vParams.begin(), m_vParams.end(), 0.0f);
```

Comment:
Where the size of m_vParams is defined? Maybe it makes sense to fill it with zeroes in the same place?